modules/models/src/main/java/org/eclipse/xpanse/modules/models/serviceconfiguration/ServiceConfigurationChangeRequestDetails.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.serviceconfiguration;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import java.util.Map;
+import java.util.UUID;
+import lombok.Data;
+import org.eclipse.xpanse.modules.models.serviceconfiguration.enums.ServiceConfigurationStatus;
+
+/**
+ * service configuration change request details.
+ */
+@Data
+public class ServiceConfigurationChangeRequestDetails {
+
+    @NotNull
+    @Schema(description = "id of service configuration update request.")
+    private UUID changeId;
+
+    @Schema(description = "name of the resource of deployed service.")
```

Comment:
name of the resource on which the change request is executed. Null means any one of the resources that is part of the service and is of type configManager can execute it and until now none of the resource have picked up this request.